base/nullable.jl

```diff
@@ -61,6 +61,68 @@ end
 
 get(x::Nullable) = isnull(x) ? throw(NullException()) : x.value
 
+"""
+    unsafe_get(x)
+
+Return the value of `x` for [`Nullable`](:obj:`Nullable`) `x`; return `x` for
```

Comment:
Okay, I'll remove the references for now. I think we will all live with the repetition in the docstring.